client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposuresAndFlights.tsx

```diff
@@ -43,76 +70,123 @@ const ExposuresAndFlights = () => {
       });
   }, [investigationId]);
 
-  const handleChangeExposureDataAndFlightsField = (fieldName: string, value: any) => {
+  const handleChangeExposureDataAndFlightsField = (index: number, fieldName: string, value: any) => {
+    const updatedExpousres = [...exposureAndFlightsData.exposures];
+    const updatedExposure = {...updatedExpousres[index], [fieldName]: value};
+    updatedExpousres.splice(index, 1, updatedExposure);
     setExposureDataAndFlights({
       ...exposureAndFlightsData,
-      [fieldName]: value,
+      exposures: updatedExpousres,
     });
   };
 
+  const onExposureAdded = (wasConfirmedExposure: boolean, wasAbroad: boolean) => {
```

Comment:
will wasConfirmedExposure and wasAbroad ever be the same? seems like they can be combined into one parameter and be the opposite of each other

Reply:
Technically they are but I thought maybe I should seperate them only for logic, but if you think its better to combine them I will

Reply:
I do think personally cause it was confusing for me but isnt that meaningful so ill resolve the thread 🤷‍♀️